Editor/AGS.Editor/Validation/Utils.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AGS.Editor.Validation
+{
+    public static class Utils
+    {
+        internal static ValidationResult Aggregate<T>(this T entity, IEnumerable<Func<T, ValidationResult>> functions)
+        {
+            return functions.Aggregate(
+                new ValidationResult(),
+                (aggr, func) =>
+                {
+                    ValidationResult next = func(entity);
+                    return new ValidationResult(
+                        aggr.Errors.Concat(next.Errors).ToArray(),
+                        aggr.Warnings.Concat(next.Errors).ToArray());
+                });
+        }
+
+        public static bool TrueForAll<T>(this T entity, IEnumerable<Predicate<T>> predicates)
```

Comment:
This and the method below doesn't appear to be referenced anywhere so we don't need to include it.